build.gradle

```diff
@@ -12,7 +12,7 @@ plugins {
 	id 'eclipse'
 	id 'idea'
 	id 'jacoco'
-	id 'org.sonarqube' version '1.2'
+	id 'org.sonarqube' version '2.1'
```

Comment:
No, but I have seem such a change in Spring Security and somewhere else.
We always can come back if our CI plan doesn't work.